From 4a70b7bd119e9600da2b94c8e7bfbc57fd9b73f5 Mon Sep 17 00:00:00 2001
From: Leo Conforti <leo@leoconforti.us>
Date: Thu, 4 May 2023 22:18:00 -0500
Subject: [PATCH 2/2] update CAMERA_MOVE_SPEED

---
 src/rcamera.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rcamera.h b/src/rcamera.h
index 0b9b8076..f13f7395 100644
--- a/src/rcamera.h
+++ b/src/rcamera.h
@@ -174,7 +174,7 @@ RLAPI Matrix GetCameraProjectionMatrix(Camera* camera, float aspect);
 //----------------------------------------------------------------------------------
 // Defines and Macros
 //----------------------------------------------------------------------------------
-#define CAMERA_MOVE_SPEED                               0.09f
+#define CAMERA_MOVE_SPEED                               3.0f
 #define CAMERA_ROTATION_SPEED                           0.03f

 // Camera mouse movement sensitivity
--
2.37.1 (Apple Git-137.1)

